okhttp/src/main/java/com/squareup/okhttp/OkHttpClient.java

```diff
@@ -120,6 +120,10 @@
         connection.connectAndSetOwner(client, owner, request);
       }
 
+      @Override public TlsVersion tlsVersionForJavaName(String name) {
+        return TlsVersion.forJavaName(name);
```

Comment:
I think we want to make `forJavaName` public both for TlsVersion and also for `CipherSuite`. Turns out there's lots of code that needs to convert between string and enum.